src/components/ServiceList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import FilterList from "../components/FilterList";
+import useServices from "../hooks/useServices";
+import ServiceCard from "../components/ServiceCard";
+
+const ServiceList = () => {
+  const apiResponse = useServices();
+  const serviceItems = apiResponse[0];
+  const isLoading = apiResponse[1];
+
+  return isLoading ? (
+    <p>Loading Baltimore County services...</p>
+  ) : (
+    <div className="container">
+      <div className="row">
+        <div className=" col-md-8 col-sm-12">
```

Comment:
this div is causing some issues with layout, is there something I am missing here on why this is included? I was expecting something more like this.

![image](https://user-images.githubusercontent.com/1933400/72370787-f3d2d700-36d0-11ea-9573-711183b8a629.png)

Reply:
nope that should be removed

Reply:
the only think i really think you need here is the row that is a parent to the items, but again, maybe i'm wrong.

Reply:
pulling out "container" div expand cards across the entire page. Now maybe thats fine since this will be placed inside of a container on the page and that will handle shrinking them? Im fine with cutting it down to just this:
 <div className="row">
            <FilterList items={serviceItems} renderItem={ServiceCard} />
          </div>

Reply:
I think you will see different results once you refactor the bootstrap out of the ServiceCard. Ideally component knows nothing about bootstrap and responds to it's container. In this case, a row would span the entire page. The site will contain a grid that this will shrink to, once in place.  

Note: This is all until we add a filters section on the left, where we will have to include a grid of some sort and this will change. However, right now this is just enough to get it "working"

Reply:
done